### PR TITLE
[New Rule] New Port Forwarding Rule Added

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/629

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_port_forwarding_added_registry.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/11/25"
+maturity = "production"
+updated_date = "2020/11/25"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a new Port forwarding rule, an adversary may use this technique to bypass network
+segmentation restrictions.
```
Comment (bm11100) [inline rules/windows/defense_evasion_port_forwarding_added_registry.toml:10]:
```suggestion
Identifies the creation of a new port forwarding rule. An adversary may abuse this technique to bypass network
segmentation restrictions.
```

File: rules/windows/defense_evasion_port_forwarding_added_registry.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "New Port Forwarding Rule Added"
+references = [
+    "https://www.fireeye.com/blog/threat-research/2019/01/bypassing-network-restrictions-through-rdp-tunneling.html",
+]
+risk_score = 43
```
Comment (bm11100) [inline rules/windows/defense_evasion_port_forwarding_added_registry.toml:20]:
```suggestion
risk_score = 47
```

File: rules/windows/defense_evasion_port_forwarding_added_registry.toml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+author = ["Elastic"]
+description = """
+Identifies the creation of a new Port forwarding rule, an adversary may use this technique to bypass network
+segmentation restrictions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "New Port Forwarding Rule Added"
```
Comment (bm11100) [inline rules/windows/defense_evasion_port_forwarding_added_registry.toml:16]:
```suggestion
name = "Port Forwarding Rule Addition"
```

**Review by bm11100 — APPROVED** [review]

**Review by devonakerr — APPROVED** [review]
RDP tunneling detection at the endpoint!

**Pull request merged**